modules/access-control/tsconfig.spec.json

```diff
@@ -0,0 +1,23 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "outDir": "dist",
+    "types": ["jest", "node", "@testing-library/jest-dom", "@testing-library/jest-dom/jest-globals"]
+  },
+  "extends": "./tsconfig.json",
+  "include": [
```

Comment:
Shouldn't this be excludes?

Reply:
`tsconfig.spec.json` is meant for test files :)

Reply:
Hehe.. Ack :)

Reply:
Hehe.. Ack :)

Reply:
Hehe.. Ack :)